code/game/gamemodes/bloodsuckers/bloodsucker.dm

```diff
@@ -0,0 +1,88 @@
+/datum/game_mode/bloodsucker
+	name = "bloodsucker"
+	config_tag = "bloodsucker"
+	report_type = "Bloodsucker"
+	antag_flag = ROLE_BLOODSUCKER
+	false_report_weight = 10
+	restricted_jobs = list("AI", "Cyborg")
+	protected_jobs = list(
+		"Captain", "Head of Personnel", "Head of Security",
+		"Warden", "Security Officer", "Detective", "Brig Physician",
+	)
+	required_players = 0
```

Comment:
Minpop needs to be at least that of vamps, probably higher

Reply:
@adamsong

Reply:
don't think it needs to be higher since we removed clans and also deals with scaling so at 15 pop there'll be like 2

Reply:
One bloodsucker can kill everybody in a 15 pop server

Reply:
i'm porting the reworked stuff on fixes so that shouldn't happen anymore